docs/tutorials/first_app/first_program.md

```diff
@@ -1,14 +1,12 @@
-## Your first Arm Mbed application
```

Comment:
@sg-

Reply:
@yennster Could you please help me create an intro paragraph? Or sit with me and verbalize what the user should expect to get out of this? Thanks so much.

Reply:
@AnotherButler Do you still need an intro paragraph before `## Choose your environment`?

Reply:
@yennster Yes. I was waiting to write it until we knew whether we'd be including Mbed CLI, but it may not be a bad idea to start drafting something that we could use whether or not we include CLI.